src_py/colordict.py

```diff
@@ -54,6 +54,7 @@
     "blue2": (0, 0, 238, 255),
     "blue3": (0, 0, 205, 255),
     "blue4": (0, 0, 139, 255),
+    "blue5": (50, 75, 245),
```

Comment:
for the colors that we add into colordict, I think we should include the `alpha` value (4th value per color) to stay consistent with the other colors already there

Reply:
Completely agreed, thank you - excellent catch!